Sources/Clappr_iOS/Classes/Animation/SeekBubble/SeekBubble.swift

```diff
@@ -3,7 +3,7 @@ class SeekBubble: UIView {
     private var images: [UIImageView] = []
     private var text: String!
     
-    private var parentView: UIView!
+    private weak var parentView: UIView!
```

Comment:
Same idea as above.

Reply:
Actually this one is already a view and it needs the reference to the parentView, since we're doing things this way with this specific plugin.
The thing that causes the retain cylce here is that in the view hierarchy, the superview has a reference to this view and vice-versa, that's why the reference must be a weak one.

If you have suggestions on how to overcome this by other means, please let me know.